content/en/docs/setup/version-skew-policy.md

```diff
@@ -0,0 +1,112 @@
+---
+reviewers:
+- sig-api-machinery
+- sig-architecture
+- sig-cli
+- sig-cluster-lifecycle
+- sig-node
+- sig-release
+title: Kubernetes Version and Version Skew Support Policy
+content_template: templates/concept
+weight: 70
+---
+
+{{% capture overview %}}
+This document describes the maximum version skew supported between various Kubernetes components.
+Specific cluster deployment tools may place additional restrictions on version skew.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Supported versions
+
+The Kubernetes project maintains release branches for the most recent three minor releases,
+and expects users to be running those versions in production.
+
+Applicable bug and security fixes may be backported to those three release branches, depending on severity and feasibility.
+Patch releases are cut from those branches at a regular cadence,
+or on an as-needed basis as determined by the [patch release manager](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/patch-release-manager/README.md#release-timing).
+
+With minor releases happening approximately every three months,
+that means each minor release is maintained for approximately nine months.
```

Comment:
do we need to mention what wg-lts is working on?

Reply:
I would not mention that here. 

This is attempting to capture the current state of the world, and be a user guide for how clusters should currently be deployed/upgraded. 

If concrete proposals by wg-lts are accepted, this document should be updated at that point.

Reply:
+ 1 to update doc when any conclusion form wg-lts :)

Reply:
FTR I agree on not mentioning it.  They're almost orthogonal.  Any highly theoretical future proposal on a change to support strongly depends on us all agreeing first on what is supported today.  This definition of present status should be concretely what we do today no more and not be muddled by discussion of change.